chunk/src/main/java/org/duracloud/chunk/FileChunkerDriver.java

```diff
@@ -215,9 +210,10 @@ private static void die() {
     /**
      * Main
      *
+     * Temporarily disabled while shuffling classes
```

Comment:
I still need to figure out what to do with this. The requirement on the ContentStoreManagerImpl makes it tricky to deal with. I was thinking it _could_ be moved to the `storeclient` module and built separately there.

Reply:
I'm not understanding what the problem is here.  Can you clarify what the issue is?

Reply:
When moving the classes around a few changes happened:
* A `storeclient-api` module was created to hold the interfaces (e.g. `ContentStoreManager`) and domain models for the client
* The `chunk` pom was updated to use the `storeclient-api` instead of `storeclient`
* The `storeclient` pom was updated to include `chunk`
* This resulted in the `ContentStoreManagerImpl` no longer being accessible in the `chunk` module, and this being disabled.

I'm wondering if maybe we want to create a `chunk-driver` which could use the `chunk` and `storeclient` as dependencies.